Source/ProgressionSystemRuntime/ProgressionSystemRuntime.Build.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Valeriy Rotermel and Yevhenii Selivanov
+
+using UnrealBuildTool;
+
+public class ProgressionSystemRuntime : ModuleRules
+{
+	public ProgressionSystemRuntime(ReadOnlyTargetRules Target) : base(Target)
+	{
+		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
+		CppStandard = CppStandardVersion.Latest;
+		bEnableNonInlinedGenCppWarnings = true;
+
+		PublicDependencyModuleNames.AddRange(new[]
+			{
+				"Core"
+				, "Bomber" // Is included in header files
+				, "UMG"
+			}
+		);
+
+		PrivateDependencyModuleNames.AddRange(new[]
+			{
+				"CoreUObject", "Engine", "Slate", "SlateCore", "MyUtils" // Core
```

Comment:
On cooking the game it shows next warning:
```
PackagingResults: Warning: Plugin 'ProgressionSystem' does not list plugin 'MyEditorUtils' as a dependency, but module 'ProgressionSystemRuntime' depends on 'MyUtils'.
```
Every module (as yours) requires put additionally the same dependency to external plugin (as MyEditorUtils) also to your .uplugin.

Please open your UProgressionSystem.uplugin as add next:
```
"Plugins": [
		{
			"Name": "MyEditorUtils",
			"Enabled": true
		}
```

_See example in NewMainMenu.uplugin_

Reply:
added dependency on an external plugin